src/messaging/tests/TestChannel.cpp

```diff
@@ -50,6 +50,7 @@ using namespace chip::Messaging;
 using TestContext = chip::Test::MessagingContext;
 
 TestContext sContext;
+Transport::DummyMessageCounterManager gDummyMessageCounterManager;
```

Comment:
Don't we need to include the right header here?

Reply:
done